### PR TITLE
Fixes errors with email validation script

### PR DESCRIPTION
**Pull request by EthanJStark**
-Fixes bug in which homepage was logging errors due to email-validation.js being called on layout.pug and login-layout.pug

### CODE REVIEW

**Comment by jrob8577** [comment]
@EthanJStark Not sure I like embedding the js files in the non-layout view.  Are we using the correct layouts for these pages?

**Comment by EthanJStark** [comment]
@jrob8577 No, the layouts being used were a bit mucked up. Fixed it.

File: views/layouts/login-layout.pug [hunk 1]
```diff
@@ -27,10 +27,6 @@ html
           .navbar-right
             a.btn.btn-primary.navbar-btn(href='/accounts/register', role='button') Sign Up
 
-
-    .container
```
Comment (jrob8577) [inline views/layouts/login-layout.pug]: @EthanJStark Did we intend to delete this?

Comment (jrob8577) [inline views/layouts/login-layout.pug]: Oh nvm I see - the layout naming is a little misleading now though.  Any thoughts on a way to change it to clarify purpose of layout?

Comment (EthanJStark) [inline views/layouts/login-layout.pug]: That's true, @jrob8577 . . . what about `email-form-layout.pug`?

Comment (jrob8577) [inline views/layouts/login-layout.pug]: Works for me.

**Comment by EthanJStark** [comment]
@jrob8577 Updated with email-form-layout change

**Pull request merged**